src/presentation/http/http-api.ts

```diff
@@ -353,6 +353,11 @@ export default class HttpApi implements Api {
         this.log.error(error);
         void reply.domainError(error.message);
       }
```

Comment:
should we `return` here?

Reply:
I think no. reply.domainError() is enough. There is no return in the official documentation in such a situation (https://fastify.dev/docs/latest/Reference/Server/#seterrorhandler)

Reply:
are you sure that `throw error` won't be called?

Reply:
I tested some cases. We need to use return